src/course-unit/data/slice.js

```diff
@@ -87,6 +89,15 @@ const slice = createSlice({
         fetchUnitLoadingStatus: payload.status,
       };
     },
+    fetchCourseVerticalChildrenDataSuccess: (state, { payload }) => {
```

Comment:
The name should reflect to the action that happens inside. I guess the appropriate name will be `updateCourseVerticalChildren`.

Reply:
Agree, changed